README.md

```diff
@@ -10,9 +10,7 @@
 Repository to describe, develop, document and test the Identity And Consent Management API family
```

Comment:
```suggestion
Repository to describe, develop, document and test the Identity And Consent Management for CAMARA APIs
```
Consistent with Commonalities. If "describe, develop, document and test the Identity And Consent Management" is correct I'm not able to judge. Test cases could make sense?